searx/static/themes/simple/src/less/definitions.less

```diff
@@ -227,6 +230,9 @@
   --color-toolkit-loader-borderleft: rgba(0, 0, 0, 0);
   --color-doc-code: #ddd;
   --color-doc-code-background: #4d5a6f;
+  // Favicons Colors
+  --color-favicon-background-color: #bbb;
+  --color-favicon-border-color: #333;
```

Comment:
Is there a reason as to why we're changing the background colours in light and dark mode?

Reply:
I found the background of transparent favicons (and the border) a bit too bright in dark mode ... so I thought it would be good to include this background and the border into the `definitions.less`.  I then made the two colors less bright in the dark theme.  If you think the latter is not good we can use the same colors in light and dark theme .. but we should not remove theses both two colors from the `definitions.less`.

Reply:
I'll leave this at your discretion - but I think the same colors should be used on both themes. There's not enough contrast between the favicon and background on the dark theme in my opinion.

Reply:
No problem .. now we have same colors in dark & light theme / [diff](https://github.com/searxng/searxng/compare/bbfd9249ef47e5afc7ebd7aca5d1300606959f69..388cdcad2c94ad0d05fb6757125ffc15d9c7272f)